packages/orchestration/src/chain-info.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * @file static declaration of known chain types will allow type support for
```

Comment:
I'd expect this as a test fixture, but not a source module.

This info will vary between a3p, devnet, mainnet, etc.

oh. weird. this is just a .ts->.js rename. it was already here.

Reply:
yeah, I figure it's all prototype code still. though the `.ts` was less risky to have in `src` because it would never show up in a bundle.

can we leave this here until #8879? I'll put a TODO